lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AlfrescoApiService } from '@alfresco/adf-core';
+import { ContentPagingQuery, Hold, LegalHoldApi } from '@alfresco/js-api';
+import { Injectable } from '@angular/core';
+import { Observable, from, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LegalHoldService {
+    private _legalHoldApi: LegalHoldApi;
```

Comment:
Sometimes we do that in case when we have getters/setters. Then we can't name field like getter because of name conflicts so we prefix that with _. In all other cases we don't use _.

Reply:
so we can leave it as it is in this case